com.viewizard.AstroMenace.json

```diff
@@ -0,0 +1,65 @@
+{
+  "app-id": "com.viewizard.AstroMenace",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "1.6",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "astromenace",
+  "rename-appdata-file": "astromenace.appdata.xml",
+  "rename-desktop-file": "astromenace.desktop",
+  "rename-icon": "astromenace",
+  "finish-args": [
+    "--socket=x11",
+    "--socket=pulseaudio",
+    "--device=dri"
+  ],
+  "modules": [
+    "shared-modules/glu/glu-9.0.0.json",
+    "shared-modules/SDL/SDL-1.2.15.json",
+    {
+      "name": "freealut",
```

Comment:
This [fails on aarch64](https://flathub.org/builds/#/builders/6/builds/986/steps/4/logs/stdio) since I think this was released with tooling that doesn't know aarch64 exists. Looks like you need to rm-configure and then autoreconf like so: https://github.com/flathub/com.github.philip_scott.spice-up/blob/11b340b49568fb976045108afaf860ca2446e50f/com.github.philip_scott.spice-up.json#L54

Reply:
fixed